store/dataStore.ts

```diff
@@ -30,4 +65,26 @@ export const useDataStore = create<DataStore>((set) => ({
       }),
     ),
   resetUser: () => set({ user: initialState.user }),
+  updateNFLTeams: (teamName, teamLogo) =>
+    set(
+      produce((state: IDataStoreState) => {
+        state.NFLTeams.teamName = teamName;
+        state.NFLTeams.teamLogo = teamLogo;
+      }),
+    ),
+    updateUserWeeklyPick: (id, weekNumber) => 
```

Comment:
m: Add prop types to this function. Make it into a deconstructed params.

Reply:
Updated:

`  updateUserWeeklyPick: ({ userId, weekNumber }: IUserWeeklyPick): void =>
    set(
      produce((state: IDataStoreState) => {
        state.user.id = userId;
        state.userWeeklyPick.weekNumber = weekNumber;
      }),`